### PR TITLE
historylist

### PR DESCRIPTION
**Pull request by marsyang1991**

### CODE REVIEW

**Comment by saukymo** [comment]
:+1:

File: app/src/main/java/cn/ac/ict/yxd/itug/Util/HistoryItemAdapter.java [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+            listItemView.iv = (ImageView)convertView.findViewById(R.id.iv_item_type);
+            listItemView.cb = (CheckBox)convertView.findViewById(R.id.cb_item);
+            //设置控件集到convertView
+            convertView.setTag(listItemView);
+        } else {
+            listItemView = (ListItemView) convertView.getTag();
+        }
+
+        RecordType type = RecordType.valueOf((String) listItems.get(position).get("type"));
+        switch(type)
```
Comment (saukymo) [inline app/src/main/java/cn/ac/ict/yxd/itug/Util/HistoryItemAdapter.java]: Better to use Map<type, icon>.
icon = Map.get(type);
listItemView.iv.setImageResource(icon);

File: app/src/main/java/cn/ac/ict/yxd/itug/Util/HistoryItemAdapter.java [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+                listItemView.iv.setImageResource(R.drawable.ic_sound_50dp);
+                break;
+            case tapping:
+                listItemView.iv.setImageResource(R.drawable.ic_finger_50dp);
+                break;
+            case cognition:
+                listItemView.iv.setImageResource(R.drawable.ic_cognition_50dp);
+                break;
+        }
+        listItemView.name.setText((String) listItems.get(position).get("name"));
```
Comment (saukymo) [inline app/src/main/java/cn/ac/ict/yxd/itug/Util/HistoryItemAdapter.java]: Better to use Map<type, icon>.
icon = Map.get(type);
listItemView.iv.setImageResource(icon);

**Comment by marsyang1991** [comment]
👍 

[Approved with PullApprove](https://pullapprove.com/yubozu/Parkinson_Health/pull-request/14/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=yubozu/Parkinson_Health)

**Pull request merged**